hrm-service/service-tests/contacts.test.ts

```diff
@@ -164,17 +173,28 @@ const deleteCsamReportsByContactId = (contactId: number, accountSid: string) =>
     `),
   );
 
+// eslint-disable-next-line @typescript-eslint/no-shadow
+const deleteReferralsByContactId = (contactId: number, accountSid: string) =>
+  db.task(t =>
+    t.manyOrNone(`
+      DELETE FROM "Referrals"
+      WHERE "contactId" = ${contactId} AND "accountSid" = '${accountSid}';
+    `),
+  );
+
 beforeAll(async () => {
   await mockingProxy.start();
   await mockSuccessfulTwilioAuthentication(workerSid);
   await cleanupCsamReports();
+  await cleanupReferrals();
```

Comment:
I think referrals has a CASCADE DELETE on it's relationship with contacts, so you might not have to delete it.

But it does no harm